central/complianceoperator/v2/report/manager/complianceReportgenerator/report_gen.go

```diff
@@ -0,0 +1,24 @@
+package complianceReportgenerator
+
+import (
+	"context"
+
+	checkResults "github.com/stackrox/rox/central/complianceoperator/v2/checkresults/datastore"
+	"github.com/stackrox/rox/pkg/notifier"
+)
+
+// ReportGenerator interface is used to generate compliance report and send email notification.
+//
+//go:generate mockgen-wrapper
+type ComplianceReportGenerator interface {
```

Comment:
Assuming manager changes to invoke the processing functionality are a separate PR?